planning/behavior_path_planner/autoware_behavior_path_lane_change_module/test/test_lane_change_scene.cpp

```diff
@@ -0,0 +1,228 @@
+
+    std::string lc_dir{"autoware_behavior_path_lane_change_module"};
+    const auto lc_param = get_absolute_path_to_config(lc_dir, "lane_change.param.yaml");
+
+    autoware::test_utils::updateNodeOptions(
+      node_options, {common_param, nearest_search_param, vehicle_info_param, bpp_param,
+                     drivable_area_expansion_param, scene_module_manager_param, lc_param});
+    return node_options;
+  }
+
+  [[nodiscard]] RouteHandlerPtr init_route_handler() const
+  {
+    std::string autoware_route_handler_dir{"autoware_route_handler"};
+    std::string lane_change_right_test_route_filename{"lane_change_test_route.yaml"};
+    std::string lanelet_map_filename{"2km_test.osm"};
+    const auto lanelet2_path =
+      get_absolute_path_to_lanelet_map(test_utils_dir, lanelet_map_filename);
+    const auto map_bin_msg = autoware::test_utils::make_map_bin_msg(lanelet2_path, 5.0);
+    auto route_handler_ptr = std::make_shared<RouteHandler>(map_bin_msg);
+    const auto rh_test_route =
+      get_absolute_path_to_route(autoware_route_handler_dir, lane_change_right_test_route_filename);
+    route_handler_ptr->setRoute(autoware::test_utils::parse_lanelet_route_file(rh_test_route));
+
+    return route_handler_ptr;
+  }
+
+  static std::shared_ptr<nav_msgs::msg::Odometry> set_odometry(const Pose & pose)
+  {
+    nav_msgs::msg::Odometry odom;
+    odom.pose.pose = pose;
+    return std::make_shared<nav_msgs::msg::Odometry>(odom);
+  }
+
+  void set_previous_approved_path()
+  {
+    normal_lane_change_->prev_module_output_.path = create_previous_approved_path();
+  }
+
+  [[nodiscard]] PathWithLaneId create_previous_approved_path() const
+  {
+    const auto & common_data_ptr = get_common_data_ptr();
+    const auto & route_handler_ptr = common_data_ptr->route_handler_ptr;
+    lanelet::ConstLanelet closest_lane;
+    const auto current_pose = planner_data_->self_odometry->pose.pose;
+    route_handler_ptr->getClosestLaneletWithinRoute(current_pose, &closest_lane);
+    const auto backward_distance = common_data_ptr->bpp_param_ptr->backward_path_length;
+    const auto forward_distance = common_data_ptr->bpp_param_ptr->forward_path_length;
+    const auto current_lanes = route_handler_ptr->getLaneletSequence(
+      closest_lane, current_pose, backward_distance, forward_distance);
+
+    return route_handler_ptr->getCenterLinePath(
+      current_lanes, 0.0, std::numeric_limits<double>::max());
+  }
+
+  void TearDown() override
+  {
+    normal_lane_change_ = nullptr;
+    lc_param_ptr_ = nullptr;
+    planner_data_ = nullptr;
+    rclcpp::shutdown();
+  }
+
+  LCParamPtr lc_param_ptr_;
+  std::shared_ptr<NormalLaneChange> normal_lane_change_;
+  std::shared_ptr<PlannerData> planner_data_ = std::make_shared<PlannerData>();
+  LaneChangeModuleType lc_type_{LaneChangeModuleType::NORMAL};
+  Direction lc_direction_{Direction::RIGHT};
+  std::string name = "test_lane_change_scene";
+  std::string test_utils_dir{"autoware_test_utils"};
```

Comment:
```suggestion
  std::string name_{"test_lane_change_scene"};
  std::string test_utils_dir_{"autoware_test_utils"};
```

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/9090/commits/0e362e625dead16c68378d8254b438b9e79c2799